src/plugins/blocksmith/components/imgur-post-block-modal/getImgurPostBlockId.js

```diff
@@ -1,7 +1,7 @@
 const getImgurPostBlockId = (str) => {
-  const IMGUR_POST_BLOCK_HAS_TYPE = /\/(a|gallery)+\//g;
-  const IMGUR_POST_BLOCK_EMBED_REGEX = /data-id="(a|gallery)?\/?\w+/g;
-  const IMGUR_POST_BLOCK_URL_REGEX = /http?s:\/\/imgur\.com\/(a|gallery)?(\w+)(\/?)\w+(\/?)/g;
+  const IMGUR_POST_BLOCK_HAS_TYPE = /\/(a|gallery)+\//;
+  const IMGUR_POST_BLOCK_EMBED_REGEX = /data-id="(a|gallery)?\/?\w+/;
```

Comment:
@cohencodes these don't need to be redefined every function call.  move them out of hte function

Reply:
@gdbrown done